tests/boards/intel_s1000_crb/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
-  boards.s1000_crb:
+  boards.s1000_crb.s1000_crb:
```

Comment:
One more duplicate. What do you think about my idea to add for each duplicate `_test` in the end like that `boards.s1000_crb.s1000_crb_test:` ?